src/Microsoft.DotNet.Build.Tasks.Feed/project.json

```diff
@@ -5,9 +5,10 @@
     "System.Security.Cryptography.Algorithms": "4.2.0",
     "System.Reflection.Metadata": "1.3.0",
     "NETStandard.Library": "1.6.0",
-    "Newtonsoft.Json": "9.0.1",
-    "NuGet.Versioning": "4.3.0-preview2-4095",
-    "NuGet.Packaging": "4.3.0-preview2-4095"
+    "Newtonsoft.Json": "10.0.3",
+    "NuGet.Versioning": "4.3.0",
+    "NuGet.Packaging": "4.3.0",
+    "SleetLib": "2.2.7"
```

Comment:
Please make sure this dependency ends up in the package dependency for the feed package.

Also do remember that given we need to build all this from source we will need to make sure Sleet conforms to being built from source in our overall efforts.

Reply:
As per our offline conversation we'll leave the package dependency instead of the DLL. I've updated the nuspec

Reply:
I thought we decided to just include the dll in the Feed package instead of adding the package dependency.